email_totals/app.py

```diff
@@ -0,0 +1,204 @@
+from email_totals import ce, org, ses
+
+import logging
+import os
+from datetime import datetime
+
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
+
+
+def report_periods(today):
+    """
+    Calculate the time periods for cost explorer.
+
+    This lambda will run at the beginning of the month, looking at the
+    previous month and comparing change to the month before that.
+
+    The Start date is inclusive, and the End date is exclusive
```

Comment:
might be nice to include this info in the docs?

Reply:
do you mean adding it to the README? because this seems more like an implementation detail of interacting with the cost explorer api to me, not really information a user needs to have to interact with the lambda

Reply:
yes README.  Yes, i can agree with you on this.